node-support/src/command-helper.js

```diff
@@ -49,7 +43,12 @@ module.exports = class CommandHelper {
    * @private
    */
   handleCommand(command) {
-    const ctx = this.createContext(command.id);
+    let metadata = new Metadata([]);
+    if (command.metadata && command.metadata.entries) {
```

Comment:
Perhaps superfluous, but would it not make sense to check that the length of the entries are > 0?

Reply:
It will never be an empty array - it will be undefined if the array is empty, since when a sequence in protobuf is empty, nothing gets encoded, and then when protobufjs decodes that, the result is that it never sets the entries property.